js/main.js

```diff
@@ -12,3 +12,25 @@ function updatePhoto(event) {
 }
 
 urlInput.addEventListener('input', updatePhoto);
+
+var form = document.querySelector('#form');
+var title = document.querySelector('#title-text');
+var notes = document.querySelector('#notes-text');
+
+function submitListener(event) {
+  event.preventDefault();
+  data.entries.unshift({
+    title: title.value,
+    photoURL: urlInput.value,
+    notes: notes.value,
+    entryID: data.nextEntryId
+  });
+  data.nextEntryId++;
+
+  photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  title.value = '';
+  urlInput.value = '';
+  notes.value = '';
```

Comment:
I would just use the `reset` method

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset